app/views/part/show.scala.html

```diff
@@ -140,7 +144,11 @@
 
     <dl>
         <dt>@Messages("parts.cache.ttl")</dt>
-        <dd>@part.cacheTtlStr</dd>
+        <dd>@part.cacheTtlStr @if(part.config.cacheTtl != Some(scala.concurrent.duration.Duration.Zero)) {
```

Comment:
Would prefer using `!part.config.cacheTtl.contains(scala.concurrent.duration.Duration.Zero)` here for a bit more type-safety when refactoring.